engine/glfw/lib/win32/win32_window.c

```diff
@@ -1526,10 +1527,18 @@ int _glfwPlatformGetDefaultFramebuffer( )
 //========================================================================
 // Set the window title
 //========================================================================
+#define MAX_WINDOW_TITLE_LENGTH 255
 
 void _glfwPlatformSetWindowTitle( const char *title )
 {
-    (void) SetWindowText( _glfwWin.window, title );
+    int title_len = (int)strlen(title);
+    wchar_t unicode_title[MAX_WINDOW_TITLE_LENGTH];
+    int res = MultiByteToWideChar(CP_UTF8, 0, title, title_len, &unicode_title, MAX_WINDOW_TITLE_LENGTH);
```

Comment:
A possible improvement is to call MultiByteToWideChar without an output buffer to get the length of the UTF8 string. We could then allocate a wchar_t buffer long enough to hold the title, set it and then free the buffer. But the question is if it is worth it?

Reply:
I think it's ok here, since we already have a max length anyways.